Developers/Unity Project/Spooksville/Assets/Scripts/Items/WeaponManager.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class WeaponManager : MonoBehaviour
+{
+    public static WeaponManager instance;
+    private List<Weapon> weapons = new List<Weapon>();
+
+    public Weapon GetRandomWeapon()
+    {
+        return weapons[Mathf.RoundToInt(Random.Range(0, weapons.Count - 1))];
```

Comment:
In general I find unity's built-in randomness and math stuff awkward unless you're using floats. Thankfully .NET provides a nicer implementation.
```suggestion
        return weapons[Mathf.RoundToInt((new Random()).Next(0, weapons.Count - 1))];
```
Also, in general you'll get worse results with a pseudorandom number generator if you add custom munging to the output afterwards. Better to use tried-and-true algorithms for producing numbers in a certain range.